### ISSUE TITLE
Freezes on FMV - Tenchu San Portable (ULJM05505)

### ISSUE THREAD

**Issue by mvergaraq** [hrydgard/ppsspp#13813]
### What happens?
When pressing Start Game and selecting Rikimaru, the game freezes into a white screen. Every other character works fine, but with Rikimaru it is unplayable probably due to a FMV that freezes the game after the loading screen.
Tried Vulkan, OpenGL, and different settings but game freezes anyway.

### What *should* happen?
Game should play a FMV and then continue as normal.

### What hardware, operating system, and PPSSPP version?
PPSSPP_v1.10.3-1380-g98e82c5c6, iOS 14.2 iPhone Xs (I know it's an unsupported device but so far every bug I've experienced on an iOS release has been also present on Android, so I'm pretty sure this needs a quick check as it may be present on more devices)

**Comment by Panderner** [comment]
This game works fine for me running on POCO M3 in OpenGL and Vulkan:
![Screenshot_2020-12-26-15-12-51-455_org ppsspp ppsspp](https://user-images.githubusercontent.com/37503397/103147144-21ebe280-478d-11eb-990b-540dab728542.jpg)
![Screenshot_2020-12-26-15-13-42-665_org ppsspp ppsspp](https://user-images.githubusercontent.com/37503397/103147146-244e3c80-478d-11eb-99be-93dcb4fa08d8.jpg)

**Comment by sum2012** [comment]
@mvergaraq Are you setting is default except IR ?
UI Lang also change to japan ?

**Comment by mvergaraq** [comment]
> @mvergaraq Are you setting is default except IR ?
> UI Lang also change to japan ?

Yes, tried with japanese as UI Lang and IR Interpreter but same bug, game freezes into a white screen when starting new game with Rikimaru

**Comment by sum2012** [comment]
Sorry,I don't have idea

**Comment by unknownbrackets** [comment]
Maybe check the CRC and verify the disc is good?

-[Unknown]

**Comment by sum2012** [comment]
@mvergaraq  Please test v1.10.3-1550-g5d8b2ef50 or later version

**Comment by mvergaraq** [comment]
@sum2012 Thank you, I've just tested that version but I still get a white screen.
Device: iPhone Xs iOS 14.2

**Comment by IrfanH495** [comment]
@mvergaraq Did you use a cheat
![Screenshot_20210214-132134](https://user-images.githubusercontent.com/78854859/107870281-46308980-6ec9-11eb-9697-dd44d490b94f.png)
disable all cheats will be normal
![Screenshot_20210214-132440](https://user-images.githubusercontent.com/78854859/107870312-99a2d780-6ec9-11eb-96a1-e8d5a170a57e.png)

**Comment by hrydgard** [comment]
Might be fixed in recent development builds, please check.
